webrtc.html

```diff
@@ -7040,6 +7041,26 @@ <h2>Dictionary <code><a>RTCRtpEncodingParameters</a></code>
               the user agent MUST still respect the limit imposed by any
               "maxptime" attribute, as defined in [[!RFC4566]], Section 6.</p>
             </dd>
+            <dt><dfn data-idl><code>adaptivePtime</code></dfn> of type <span class="idlMemberType">boolean</span>, defaulting to <code>false</code>.</dt>
+            <dd>
+              <p>Indicates whether this encoding MAY dynamically change
+              the frame length. If the value is <code>true</code>,
+              the user agent MAY use any valid frame length for any of its
+              frames, and MAY change this at any time. Valid values are positive
+              integers divisible by 10. If the "maxptime" attribute (defined in
+              [[!RFC4566]] Section 6) is specified, that maximum applies.</p>
+              <p class="note">Using a longer frame length reduces the
+              bandwidth consumption due to overhead, but does so at the cost
+              of increased latency. Allowing the user agent to change the
+              frame length dynamically, allows it to adapt its bandwidth
+              allocation strategy based on the current network conditions.</p>
+              <p>The value for this configuration option cannot be changed
+              from its initial value. If this is attempted,
+              <code>InvalidModificationError</code> MUST be thrown.</p>
```

Comment:
Note: The rationale for throwing if you try to turn off adaptivePtime after-the-fact is that it may be unclear which ptime to use at that point (should it change to some default ptime or use whatever the current ptime was, etc?). If we want to allow turning this off after-the-fact in order to allow e.g. reusing senders in non-adaptive cases we could revisit this in a follow-up issue.